content/v2/openapi.yml

```diff
@@ -23,7 +23,9 @@ servers:
     variables: {}
   - url: 'https://api.sandbox.dnsimple.com/v2'
     description: >-
-      DNSimple Sandbox API. For more information on the purpose and use of the Sandbox API, see our [testing documentation](https://developer.dnsimple.com/v2/#testing)
+      DNSimple Sandbox API. For more information on the purpose and use of the
+      Sandbox API, see our [testing
+      documentation](https://developer.dnsimple.com/v2/#testing)
```

Comment:
Was the switch from 1 line to 3 lines intentional or the result of an automated tool?

Reply:
It was the export from the Swagger Editor.